pycozmo/activity.py

```diff
@@ -24,6 +27,7 @@ class BehaviorChooser:
         "choice_type",
         "behaviors",
         "iteration",
+        "__dict__",
```

Comment:
Why is this necessary?

Reply:
I added it to have as optional variables the `behavior_scores`, `behavior_names`, `behavior_repetitions` and `repetition_penaltys`. Done this way, they don't need to be defined in the slots. I thought it made sense since these variables are generated for the scored choosers only, but I can see that there are arguments for specifying them in slots directly or not using slots at all here. What do you prefer? I'm still learning to use `__slots__` vs `__dict__` to define variables in these classes (always used the default, `__dict__` before), so I would appreciate to know what would you do, and what you think makes more sense and/or is faster/more efficient.